style.css

```diff
@@ -0,0 +1,153 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+  scroll-behavior: smooth;
+}
+
+@media screen and (min-width: 374px) {
+  body {
+    background-image: url(background-1.svg);
+    background-position: center;
+    font-weight: 700;
+    font-size: 18px;
+    color: var(--primary);
+    animation: logo 2s ease-in-out;
+    text-decoration: none;
+  }
+
+  .div-container {
+    width: 375px;
+    height: 5410px;
+  }
+
+  .header {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 15vh;
+    width: 88%;
+    margin-left: 10%;
+  }
+
+  .My-Logo {
+    display: flex;
+    width: 100%;
+    color: #6070ff;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    height: 20px;
+  }
+
+  .header-Line-1 {
+    margin: 2px;
+    width: 16px;
+    border: 2px solid #acb7c3;
+  }
+
+  .header-Line-2 {
+    margin: 2px;
+    width: 16px;
+    border: 2px solid #acb7c3;
+  }
+
+  .header-Line-3 {
+    margin: 2px;
+    width: 16px;
+    border: 2px solid #acb7c3;
+  }
+
+  .section-1 {
+    display: flex;
+    justify-content: start;
+    width: 80%;
+    margin-left: 10%;
+    margin-top: 15vh;
+  }
+
```

Comment:
- I believe it will be better if we used equal margins on the left and right side of our entire `section-1` as you seem to have more space on the left and less on the right.

Here is what I mean 👇🏽 

| Your header section  |
| :---:   | 
|<img width="389" alt="Screenshot 2023-02-16 at 11 12 42" src="https://user-images.githubusercontent.com/99246359/219439715-4de0ea8f-5a66-4e76-b3cf-ae3a7544326a.png">  |